README.md

```diff
@@ -272,18 +271,18 @@ service studentData on studentServiceListener {
         _ = observe:addTagToSpan("error_counts", <string>studentData.errors);
     }
 
+    // Resource configuration for making a mock error to the system.
     @http:ResourceConfig {
         methods: ["GET"],
         path: "/testError"
     }
-    // Test Error resource is used to make a mock error.
+    // Test Error resource to make a mock error.
     resource function testError(http:Caller caller, http:Request request) {
         studentData.requestCounts += 1;
         http:Response response = new;
-
         studentData.errors += 1;
         io:println(studentData.errors);
-        // The below function adds tags that are to be passed as metrics in the traces. These tags are added to the default ootb system span.
+        // The below function adds tags that are to be passed as metrics in the traces. These tags are added to the default system span.
```

Comment:
The below function adds tags to be passed as metrics in the message traces. These tags are added to the default system span.